region/maridia/inner-green/East Sand Hall.json

```diff
@@ -679,6 +679,28 @@
         "A little higher run speed can also work but may require releasing jump near the peak to avoid getting caught on the overhang."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Cross-Room Blue Speed Jump",
```

Comment:
The name sounds a lot like the notable

Reply:
True. Created a separate notable for this now: "Cross-Room Blue Speedy Jump (Right to Left)". And renamed the existing notable "Cross-Room Speed Jump" to "Cross-Room Speed Jump over Evirs", to make the difference more clear. And a few other little clean-ups to the names. For example, I tried to remove most instances of "Left to Right" or "Right to Left" from strat names where the direction doesn't make a significant difference.